dev/release/verify-release-candidate.bat

```diff
@@ -73,9 +73,8 @@ call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\Tool
 
 @rem NOTE(wesm): not using Ninja for now to be able to more easily control the
 @rem generator used
-
-cmake -G "%GENERATOR%" ^
-      -DCMAKE_INSTALL_PREFIX=%ARROW_HOME% ^
+@rem NOTE(npr): trying without specifying a generator explicitly
+cmake -DCMAKE_INSTALL_PREFIX=%ARROW_HOME% ^
```

Comment:
I'll revert this since I "fixed" this with the UNITY build option for now. We should open a JIRA about the parquet-internals-test failure since that's not covered by ARROW-8510